### PR TITLE
(feat) remove presigned_url from drs object

### PR DESCRIPTION
**Pull request by BinamB**
Jira Ticket: [PXP-xxxx](https://ctds-planx.atlassian.net/browse/PXP-xxxx)
- [ ] Remove this line if you've changed the title to (PXP-xxxx): <title>

### New Features
- Remove presigned url from the `access_url` field in the drs object when authz is given.

### CODE REVIEW

File: indexd/drs/blueprint.py [hunk 1]
```diff
@@ -50,7 +50,6 @@ def get_signed_url(object_id, access_id):
 
 
 def indexd_to_drs(record, list_drs=False):
-    bearer_token = flask.request.headers.get("AUTHORIZATION")
```
Comment (jiaqi216) [inline indexd/drs/blueprint.py:53]: So we still have to accept the auth token but we're just going to ignore it? cc @Avantol13 if you're cool with it then I"m not picky about it either.

Comment (BinamB) [reply indexd/drs/blueprint.py:53]: Oh that's been removed .

Comment (Avantol13) [reply indexd/drs/blueprint.py:53]: yeah, that's fine by me

**Comment by jiaqi216** [comment]
Belated LGTM from me too!

**Pull request merged**